src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/singlespaceseparator/InputSingleSpaceComments.java

```diff
@@ -1,4 +1,4 @@
-package com.puppycrawl.tools.checkstyle.checks.whitespace;
+package com.puppycrawl.tools.checkstyle.checks.whitespace.singlespaceseparator;
 
 public class InputSingleSpaceComments {
```

Comment:
Inputs should be named after the test. Test is `SingleSpaceSeparatorCheckTest`, so input names should start with `InputSingleSpaceSeparator`.
Please do this for all inputs.